index.html

```diff
@@ -194,9 +294,77 @@ <h3 class="aboutme-techs-title">Skills</h3>
           
          </ul>
       </li>
+      <li class="aboutme-img-d">
+
+      </li>
     </ul>  
  </section>
- 
+
+
+ <section class="contact-form">
+
+  <div class="contact-form-d">
+    
+        <header class="contact-logo">
+          <img  src="./images/image_geometry_1.png" alt="logo image">
+        </header>
+        <div class="contact-logo1-d">
+            <img   class="contact1-img" src="./images/image-geometry_6.png" alt="logo desktop1">
+        </div>
+       <div class="contact-card">
+
+        <div class="contact-title-d">
+          <h2 class="contact-title">Get Started</h2>
+          <p class="contact-para">I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.</p>
+        </div>  
+          <form action="https://formspree.io/f/xqkovbew"
+            method="POST" class="contact">
+          <div class="input-d1">
+            <label for="name">Your name</label>
+            <input type="text" id="fname" maxlength="30"  placeholder="Enter your name" required>
+          </div>
+          <div class="input-d2">
+            <label for="email">Your email</label>
+            <input type="email" id="email" placeholder="Enter your email" required>
+          </div> 
+
+          <div class="input-d3">
+            <label for="message">Your message</label>
+            <textarea name="your message" id="" cols="30" rows="10" 
+              placeholder="Enter your message here..." maxlength="500" required></textarea>
+            <button type="submit" value="Start Collaboration" class="form-btn">Start Collaboration</button>
+          </div>  
+          </form>
+          
+       </div>
+
+      <div class="contact-logo2-d">
+        <img  class="contact-img" src="./images/image-geometry_7.png" alt="logo desktop2">
+      </div>
+
+  </div>
+
+      <div class="contact-footer">
+          <ul class="get">
+            <li><h3>Get my resume</h3></li>
+            <li><i class="fa fa-external-link remove"></i></li>
+            <li class="get3"><span> Get my  CV</span> <img src="./images/ic-link_teal.png" alt="ic_download"></li>
+          </ul>
+         
+        
+        <ul class="contact-social"> 
+          <li><a href=""><i class="fa fa-stack-overflow fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-github fa-inverse "></i></a></li>
+          <li><a href=""><i class="fa fa-twitter fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-linkedin fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-medium fa-inverse"></i></a></li>
+          <li><a href=""><i class="fa fa-angellist fa-inverse"></i></a></li>
+        </ul>
+    </div>
+  
```

Comment:
Great job on the contact section! 👍 However, upon reviewing the mobile version, I noticed an extra white space on the right side of the section. Please remove this white space to ensure a clean and well-aligned layout.

Furthermore, to align your elements as per the design and meet the project requirements, I recommend using Flexbox. Flexbox provides a flexible and efficient way to handle element alignment and spacing. By applying flexbox properties to your contact section, you can achieve the desired alignment and eliminate any inconsistencies.

Remember, paying attention to these details will enhance the overall user experience and ensure your design matches the provided specifications. Keep up the good work! 😄

![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/55840999/ce6b1b54-fe26-4e38-9312-a963413f58d4)